expert-forecast/expert-forecast-app/app.R

```diff
@@ -79,15 +98,14 @@ check_state_list <- googlesheets4::read_sheet(ss = submission_sheet,
 
 if(load_date %in% check_state_list$forecast_date){
     
-    list_states <- check_state_list$list_states
+    list_states <- check_state_list %>%
+        filter(forecast_date == load_date) %>%
+        .$list_states
     list_states <- unlist(str_split(list_states, pattern = ","))
     
 } else {
     
-    list_states <- df %>%
-        filter(q_type == "quantile0.5") %>%
-        .$state_name %>%
-        unique()
+    list_states <- states_100
     list_states <- c(sample(setdiff(list_states, "US"), 5), "US")
```

Comment:
does this mean the sample of states shown is different for everybody?  - surely not?